lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -37,6 +37,10 @@ def descriptions(version)
       end
     end
 
+    def all_questions
+      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
```

Comment:
I find the name of the block argument, `versions` confusing. I think something like `questions` or `all_questions` would be less confusing.